.github/workflows/ruby.yml

```
@@ -0,0 +1,78 @@
+name: Ruby
+
+on:
+  pull_request:
+    branches: [master,staging]
+  push:
+    branches: [master,staging]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      DB_PORT_5432_TCP_ADDR_TEST: localhost
+      DB_ENV_POSTGRESQL_USER_TEST: postgres
+      DB_ENV_POSTGRESQL_PASS_TEST: postgres
+      DB_ENV_POSTGRESQL_DB_TEST: postgres
+      RAILS_ENV: test
+
+    services:
+      postgres:
+        image: postgres:17
```

```suggestion
        image: postgres:14
```
...would be best, that's what we're running presently; but likely there's no big difference.

---

stick on version 14